app/main.py

```diff
@@ -1,3 +1,28 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    try:
+        with open("app/config.json", "r") as config:
+            config_data = json.load(config)
+    except FileNotFoundError as e:
+        print(e)
+        return
+
+    Car.fuel_price = config_data["FUEL_PRICE"]
+    customers_dict, shops_dict = config_data["customers"], config_data["shops"]
+    customers = [
+        Customer.get_customer_from_dict(customer)
+        for customer in customers_dict
+    ]
+    shops = [Shop.get_shop_from_dict(shop) for shop in shops_dict]
+
+    for customer in customers:
```

Comment:
Three for loops over the same data is a bit too much. You can reduce it to two loops for efficeincy. I would recommend to create Shops first, and then iterate over same data set to create customers, cars and do other things required